src/components/10-atoms/radio/index.scss

```diff
@@ -173,7 +173,6 @@ axa-radio {
         svg {
           width: 48px;
           height: 48px;
-          fill: currentColor;
```

Comment:
couldn't it be, that there is something wrong with the icons instead of the radio?

Reply:
No, we support `color` and `background-color` on the icons. We do not allow for custom fills, because they can completely change the appearance of an icon. I got this discussed with Thomas.